wtxmgr/tx.go

```diff
@@ -3460,7 +3464,15 @@ func (s *Store) Balance(ns, addrmgrNs walletdb.ReadBucket, minConf int32,
 	_, syncHeight := s.MainChainTip(ns)
 	switch balanceType {
 	case BFBalanceFullScan:
-		return s.balanceFullScan(ns, addrmgrNs, minConf, syncHeight, all, account)
+		bals, err := s.balanceFullScan(ns, addrmgrNs, minConf, syncHeight)
+		if err != nil {
+			return 0, err
+		}
+		bal, ok := bals[account]
+		if !ok {
+			return 0, nil
+		}
+		return bal.Total, nil
```

Comment:
no, this can't return the total.  it should probably be spendable instead.  total completely ignores the minconf parameter to this function.